### DISCUSSION TITLE
Responsiveness

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [archaiodata/thacer#17] [General]
Something to do later :
Discuss and evaluate how the site is working on smaller device. Also discuss how much effort we want to put in this issue

**Comment by AlZorba** [comment]
That is definitely an option to consider, an realistic plan would be to allow users to walk around Thasos with their smartphone and spot what was found in their location.
Can be budget in the next financial year.

**Comment by francoiscrouzet** [comment]
A quick work on this : https://github.com/archaiodata/thacer/issues/59
And for a full work on the topic, it could be then another ticket
